src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -79,6 +80,7 @@ private Properties buildProperties() {
         props.setProperty("resource.loader", "class");
         props.setProperty("class.resource.loader.class", ClasspathResourceLoader.class.getCanonicalName());
         props.setProperty("runtime.log", new File(configuration.getReportDirectory(), "velocity.log").getPath());
+        props.setProperty("eventhandler.referenceinsertion.class", "org.apache.velocity.app.event.implement.EscapeHtmlReference");
```

Comment:
The way it is currently setup that is correct. 

However it is possible to exclude some labels from being escaped. Though I would not recommend this. Rather I would suggest adding a markdown parser and use markdown formatting to provide the markup.

You can read about the functions and options that velocity provides here:

https://velocity.apache.org/engine/devel/developer-guide.html#orgapachevelocityappeventreferenceinsertioneventhandler